frontend/src/metabase/app.js

```diff
@@ -11,6 +11,8 @@ import "number-to-locale-string";
 // Should be imported before any other metabase import
 import "ee-overrides"; // eslint-disable-line import/no-duplicates
 
+import "metabase/lib/dayjs";
```

Comment:
I think @ranquild had a very similar PR which he abandoned

Reply:
@kulyk check this https://github.com/metabase/metabase/pull/34897/files#r1366806961 , initial PR is https://github.com/metabase/metabase/pull/34802/

Reply:
Yeah, I wasn't sure what he meant by "hard to manage plugins loaded globally" 🤔 
I'm seeing 5 failed E2E test groups and I'm hoping they're flakes :lol_sob:

Reply:
It broke storybook, so lets import this file also there

Reply:
@kulyk if you decided to use this approach please move other dayjs.extend to this file

Reply:
My initial idea for reverting the PR was that we can get away from using plugins globally, but if such basic functionality requires a plugin I guess that's not realistic